graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -624,11 +665,32 @@ def _compile_root_ast_to_ir(schema, ast, type_equivalence_hints=None):
     # Construct the start location of the query, and the starting context object.
     location = Location((base_start_type,))
     context = {
+        # 'tags' maps to a dict containing
```

Comment:
describe what the key is: `'tags' is a dict mapping the tag name to a dict containing...`